hazelcast/src/main/java/com/hazelcast/kubernetes/RuntimeContext.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.kubernetes;
+
+import javax.annotation.Nullable;
+
+public class RuntimeContext {
+
+    /**
+     * Unknown value
+     */
+    public static final long UNKNOWN = -1;
+
+    // desired number of replicas. Corresponds to StatefulSetSpec.replicas
+    private final int desiredNumberOfMembers;
+
+    // number of ready replicas. Corresponds to StatefulSetStatus.readyReplicas
+    private final int readyReplicas;
```

Comment:
can we use same terminology for both `desiredNumberOfMembers ` and `readyReplicas`, now it is replica vs member, seems we can consistently use one of them.

Reply:
I cleaned up terminology in `ClusterTopologyIntentTracker` and used the same terms in `RuntimeContext` also.
See javadoc of `ClusterTopologyIntentTracker` in https://github.com/hazelcast/hazelcast/commit/822fb6e093dda80aaa149090a033cddf6f0fac73